server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -521,28 +521,28 @@ export class ScribeLambda implements IPartitionLambda {
         this.protocolHead = protocolHead;
     }
 
-    private async sendSummaryAck(contents: ISummaryAck) {
+    private async sendResponse({ type, contents }: {
```

Comment:
This looks unrelated. Did this file get added accidentally?

Reply:
Yes!! Whoops.